### DISCUSSION TITLE
OPTON GP801R and RB5009UPr

### DISCUSSION THREAD

**Discussion by RobJan8** [Anime4000/RTL960x#243] [Q&A]
Hey

I have an Opton GP801R, which is similar to the V2801F. The firmware is V2801F_V1.9.0-220425.
When I used the stick in TP-LINK MC220L and cooper transciver in RB5009UPr everything was ok.
Opton stick does not work in SFP+ port of RB5009UPr so I changed via telnet speed LAN Mode from 1G to 2.5G with commands:

```
# flash get LAN_SDS_MODE
LAN_SDS_MODE=1
# flash set LAN_SDS_MODE 6
LAN_SDS_MODE=6
```

Currently the stick is not working. No access from TP-LINK MC220L - I know, because it is probably 2.5G.
I have no connection in RB5009UPr. I set the SFP+ port in 2.5Gbase-x mode but with no effect. What can I do more? How to get to the stick?

**Comment by Anime4000** [comment]
issue like this are out of luck, you need recover via UART, and some soldering skills

**Reply by RobJan8** [reply]
@Anime4000 thanks for your answer. What is the case on the WAN side? Is it possible to log into the stick from the OLT side?

**Reply by Anime4000** [reply]
that not possible without TR069
